sherpa/bin/conv_emformer_transducer_stateless2/streaming_server.py

```diff
@@ -527,6 +527,8 @@ def main():
 """
 
 if __name__ == "__main__":
+    # fmt: off
```

Comment:
I think it should be
```
# fmt: off
formatter = xxx
# fmt: on
```

I am using `fmt:off` and `fmt:on` to tell the `black` formatter to not format the code within the decorated block.

Reply:
> I suggest ignoring errors reported by flake8 in https://github.com/k2-fsa/sherpa/runs/7479612670?check_suite_focus=true
> 
> We can edit https://github.com/k2-fsa/sherpa/blob/master/.flake8 to do that. (Either ignore the whole file or ignore just some specific types of errors.)
> 
> You can use https://github.com/k2-fsa/icefall/blob/master/.flake8 as an example.

Done

Reply:
Done